tool/model/network.py

```diff
@@ -97,57 +112,41 @@ def get(self, k):
                     return address.get("locality")
             else:
                 return None
+
         if k == "coordinates":
             return self.feature.geometry().asPoint().toString()
+
         if k == "phase/name":
             phase = self.properties.get("phase", {})
-            try:
-                return phase.get("name")
-            except AttributeError:
-                phase_json = json.loads(phase)
-                return phase_json.get("name")
+            return phase.get("name")
+
         if k == "physicalInfrastructureProvider":
             # Only compare name, id is irrelevant
             pip = self.properties.get("physicalInfrastructureProvider", {})
-            try:
-                return pip.get("name")
-            except AttributeError:
-                pip_json = json.loads(pip)
-                return pip_json.get("name")
+            return pip.get("name")
+
         if k == "networkProviders":
             # Return a list of the names in the array, as other properties are irrelevant
             nps = self.properties.get("networkProviders")
```

Comment:
```suggestion
            nps = self.properties.get("networkProviders", [])
```

suggest return empty list if not present, or else the `for np in nps:` below will except